eng/Version.Details.xml

```diff
@@ -14,40 +14,32 @@
       <Sha>5ce78f66d89ea529e459abddb129ab36cb5bd936</Sha>
       <SourceBuild RepoName="diagnostics" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-externals" Version="8.0.0-alpha.1.23219.1">
+      <Uri>https://github.com/dotnet/source-build-externals</Uri>
+      <Sha>d1db09ce5c5a2aeab37d01336765861c0283f76f</Sha>
+      <SourceBuild RepoName="source-build-externals" ManagedOnly="true" />
+    </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.23225.2">
+      <Uri>https://github.com/dotnet/source-build-reference-packages</Uri>
+      <Sha>4bf273ec25cc3983d6a5991cf0a4aa082cb255a5</Sha>
+      <SourceBuild RepoName="source-build-reference-packages" ManagedOnly="true" />
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="8.0.0-preview.4.23210.1">
```

Comment:
vstest shouldn't have active flow from runtime, as it adds another expensive coherency path.

Reply:
If this is an action we have to lead on our side, I would be happy to have some more information because I don't know what you are referring to.

Reply:
Basically, vstest should just depend on a stable version of M.E.D